src/Client.ts

```diff
@@ -7,6 +7,8 @@ import { SchemaConstructor } from './serializer/SchemaSerializer';
 
 export type JoinOptions = any;
 
+export let devMode: boolean = false;
```

Comment:
variable should be at `Client` scope instead of module scope in order to support having multiple Colyseus servers that may not all use `devMode` enabled.